inbm/dispatcher-agent/dispatcher/sota/constants.py

```diff
@@ -27,6 +27,9 @@
 # Device local cache for SOTA
 SOTA_CACHE = str(INTEL_MANAGEABILITY_CACHE_PATH_PREFIX / 'repository-tool' / 'sota')
 
+#SNAPSHOT PATHS
+SNAPSHOT_DIRECTORY_PATH = '/etc/snapper/configs/'
+SNAPSHOT_FILENAME = '/etc/snapper/configs/rootConfig'
```

Comment:
Why rootConfig specifically? Is that the config we use for creating snapshots? Also, this would be a config filename, not a snapshot filename (variable is confusing).